### DISCUSSION TITLE
can't find `pmdarima` in env

### DISCUSSION THREAD

**Discussion by fkiraly** [alan-turing-institute/sktime#5029] [Q&A] [answered]
posted in an issue by @iSantPy :

> I installed pmdarima in the current venv using python -m pip install pmdarima. When I type python -m pip list I got:
> 
> ![image](https://user-images.githubusercontent.com/87784833/257919052-c7d96525-3991-4574-8a31-26e10cfbb48c.png)
> 
> so, there it is! but when I open a JupyterLab with the venv activate and run the program it says:
> 
> ![image](https://user-images.githubusercontent.com/87784833/257919321-bf61ff37-6cd3-4100-bf8c-e03d8d9c6620.png)
> 
> what am I doing wrong?

**Comment by fkiraly** [comment, marked as answer]
@isantpy, most likely (I'm only guessing) it is one of two:

* you are using a different env in the jupyter notebook
* you have not restarted the kernel after installing `pmdarima`

**Reply by iSantPy** [reply]
@fkiraly you are right! I didn't change the env in jupyterlab. Now works fine. Thanks!

**Reply by fkiraly** [reply]
ah, great!
